app/helpers/application_helper/button/basic.rb

```diff
@@ -12,6 +12,10 @@ def initialize(view_context, view_binding, instance_data, props)
     end
   end
 
+  def check_rbac?
```

Comment:
I tend to like passing in user specific attributes into these classes.
having these classes looking up user and rbac seems pretty smart.

just a comment and no need for any change